globals.nix

```diff
@@ -0,0 +1,12 @@
+self: super: {
```

Comment:
can we rename the file into `globals-deployers.nix` so that it is not be confused with a "default" `globals.nix`?